index.xml

```diff
@@ -14109,6 +14240,155 @@ C implements A;</x:codeblock>
           </ol>
         </div>
 
+        <div id='es-namespaces' class='section'>
+          <h3>Namespaces</h3>
+
+          <p>
+            For every <a class='dfnref' href='#dfn-namespace'>namespace</a> that is <a
+            class='dfnref' href='#dfn-exposed'>exposed</a> in a given ECMAScript global environment,
+            a corresponding property <span class='rfc2119'>MUST</span> exist on the ECMAScript
+            environment's global object. The name of the property is the <a class='dfnref'
+            href='#dfn-identifier'>identifier</a> of the namespace, and its value is an object
+            called the <dfn data-export='' id='dfn-namespace-object'>namespace object</dfn>.
+          </p>
+          <p>
+            The property has the attributes <span class='descriptor'>{ [[Writable]]: <span
+            class='esvalue'>true</span>, [[Enumerable]]: <span class='esvalue'>false</span>,
+            [[Configurable]]: <span class='esvalue'>true</span> }</span>. The characteristics of a
+            namespace object are described in <a href='#namespace-object'>section <?sref
+            namespace-object?></a> <?sdir namespace-object?>.
+          </p>
+
+          <div id='namespace-object' class='section'>
+            <h4>Namespace object</h4>
+
+            <p>
+              The namespace object for a given <a class='dfnref' href='#dfn-namespace'>namespace</a>
+              <var>namespace</var> and Realm <var>realm</var> is created as follows:
+            </p>
+
+            <ol class='algorithm'>
+              <li>
+                Let <var>namespaceObject</var> be
+                ! <a>ObjectCreate</a>(the <a>%ObjectPrototype%</a> of <var>realm</var>).
```

Comment:
This is using standard ES syntax. I think we just want to acclimatize people to it, as all ES algorithms across all specs are now written this way. See https://tc39.github.io/ecma262/#sec-algorithm-conventions